docs/builtins/General/chain-data.md

```diff
@@ -0,0 +1,41 @@
+## chain-data
+Use `chain-data` to retrieve transaction public metadata, returning an object with fields including 'chain-id', 'block-height', 'block-time', 'prev-block-hash', 'sender', 'gas-limit', 'gas-price', and 'gas-fee'.
```

Comment:
this should be slated for rewording. Really, it should explain that this is blockchain-specific metadata available for a transaction.